images/linux/scripts/installers/azpowershell.sh

```diff
@@ -30,68 +30,36 @@ if isUbuntu20 ; then
     exit 0
 fi
 
+# List of versiion
+if isUbuntu20 ; then

```

Comment:
Seems like a block on the top is not needed anymore

Reply:
Yes, agree. Thank you.